CleverRaven/Cataclysm-DDA
# [Magiclysm] Added crowbar use_actions to toolbars

#### Summary
Bugfixes "Added the crowbar use_action to the Technomancer's toolbar and mana-toolbar."


#### Purpose of change
Fixes #53996


#### Describe the solution
added the `"CROWBAR"` use_action to both toolbars


#### Describe alternatives you've considered
None


#### Testing
1. start a new game and spawn in both toolbars
2. open the (a)ctivate menu and see the prying actions for them


#### Additional context
You might notice that the mana-toolbar has `[ "CROWBAR", 0 ]` rather than just `"CROWBAR"`, this is to indicate that this use_action will use 0 charges/mana (seen via the % menu). Right now, it actually **_does_** use charges/mana, this is a bug that affects **_all_** items with some use_actions that are meant to use charges and some that are not. (see #54120 and #53620)

I also said I would make this PR over a month ago. sorry for the long wait.
